NotificationApi/NotificationApi.IntegrationTests/Seeding/TemplateDataSeedingTests.cs

```diff
@@ -6,46 +6,48 @@
 using NotificationApi.DAL;
 using NUnit.Framework;
 
-namespace NotificationApi.IntegrationTests.Seeding;
```

Comment:
the compiler was throwing warnings about not being within named namespaces so I added the curly braces to reduce the noise from builds